### DISCUSSION TITLE
Offcanvas accessibility question

### DISCUSSION THREAD

**Discussion by louismaximepiton** [twbs/bootstrap#37571] [Q&A]
Hey there, 

I'm quite a noob on accessibility but I wanted to have some other point of views. I feel like this question might be answered by a11y experts since it's more a question of accessibility but feel free to discuss about the subject in this thread.
I was using aXe to debug some a11y issues and I found two things overall:

- Offcanvases are labelledby their title but they don't have any specific role. My question here is: should we introduce a role on those divs ? Or maybe should offcanvases be inside `<section>`s instead of `<div>`s ? So I got this issue: 
```sh
aria-labelledby attribute is not well supported on a div with no valid role attribute.
```

- My second question is related to the Bootstrap documentation itself, I can see that the left sidebar menu (which is the main navigation of the docs) is inside an `<aside>` element. But as far as I can see, the `<aside>` element should be used to add something indirectly related to the main content (according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)). So here's my question: Is there any reason I'm missing to have this tag wrapping the main Bootstrap navigation ? Should we remove the aside element to only have the offcanvas markup (with or without a `<section>` element) ?

If any answer is yes or if you don't see what I mean by this, I could open a PR on one/both subject(s).
Thanks a lot for your time !

**Comment by gianlucajahn** [comment]
The ```<aside>``` element is used to wrap content that is indirectly related to the main content of the page. In the case of the Bootstrap 5 documentation, the left sidebar menu is directly related to the main content of the page, as it is the main navigation of the documentation. Therefore, using the ```<aside>``` element to wrap the left sidebar menu is not semantically correct.

Instead, it would be better to use a different HTML element, such as a ```<nav>``` element, to wrap the left sidebar menu. The ```<nav>``` element is specifically designed for navigation, and using it would be more semantically correct in this case.
